prisma/schema.prisma

```diff
@@ -11,15 +11,31 @@ generator client {
 }
 
 model Company {
-  companyId        Int     @default(autoincrement()) @id
+  companyId        Int   @default(autoincrement()) @id
   name     String
+  users User[]
```

Comment:
can you improve the formatting in this file?

Int
Int
String
User[]

should be on the same vertical line

Reply:
Done